src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -793,7 +800,7 @@ export const ShippingRoute: FC<ShippingProps> = props => {
                     setAddressNeedsVerification(false)
                     setAddressHasBeenVerified(true)
                     setAddress({ ...address, ...chosenAddress })
-                    onContinueButtonPressed()
+                    finalizeFulfillment()
```

Comment:
Alternatively we can use `useEffect` to wait for `addressHasBeenVerified` state to be updated. However, I decided to avoid it because [we might not need it](https://react.dev/learn/you-might-not-need-an-effect).